metadata/zoom.xml

```diff
@@ -7,12 +7,12 @@
 		<option name="modifier" type="key">
 			<_short>Zoom in the desktop by scrolling with modifier</_short>
 			<_long>Scrolls with the specified modifier to zoom in and out.</_long>
-			<default>&lt;super&gt;</default>
+			<default>&lt;super&gt; &lt;ctrl&gt;</default>
```

Comment:
Since this is an accessibility feature, it would make sense for the default to have a single modifier, so it's easier to find. My personal preference is the super key, as it stands.

Reply:
I chose it to match applications with <kbd>Control</kbd> + scroll.